app/views/admin/videos/index.html.erb

```diff
@@ -13,3 +13,28 @@
   <br>
   <%= f.submit "Submit", class: "btn btn-primary col-sm-2 col-sm-offset-5" %>
 <% end %>
+
+<div class="row">
+  <table class="table table-condensed">
+    <thead>
+    <tr class="warning">
+      <th data-field='title'>Title</th>
+      <th data-field='url'>Url</th>
+      <th data-field='updated_at'>Last Updated</th>
+      <th data-field="Edit/Delete">Edit/Delete</th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <% @videos.each do |video| %>
+      <tr id="table-row" data-title= "<%= video.id %>">
+        <td><%= video.title %></td>
+        <td><%= video.url %></td>
+        <td><%= video.updated_at.strftime("Updated on %m/%d/%Y %I:%M%p") %></td>
```

Comment:
I think this is the only place this would live tho. Im fine with or without a helper.